### ISSUE TITLE
Create mobile layout

### ISSUE THREAD

**Issue by coldspark29** [SeaDve/Mousai#24]
I think it would be great to have an improved mobile layout with bigger buttons, because this will be a perfect Shazam alternative for mobile Linux phones.

**Comment by SeaDve** [comment]
It already follow gnome's standards for mobile phones. I don't think we can do much here

**Comment by coldspark29** [comment]
I was actually thinking of a big round Mousai logo button in the middle or a slider at the bottom. The upper left corner is really the worst of all places for a mobile phone. I know Gnome is very limited in that regard. Maybe a Qt widget would be more suitable. I am actually starting to tinker with Qt Designer atm. Maybe I will build something when I find the time next year.

**Comment by SeaDve** [comment]
Ah, that makes sense, although moving the button at the buttom would require design, and not pretty easy to make it well. Making use of Qt is not an option for me too because I made this app specifically for Gnome. Of course, it would still work well with other DEs and WMs.

**Comment by apandada1** [comment]
Something like [this](https://tuxphones.com/tutorial-developing-responsive-linux-smartphone-apps-gnome-builder-gtk-libhandy-gtk-part-1/) is probably possible with libadwaita also. When the screen size is small, the `listen` button can go to the bottom, so that it would be easily accessible with the thumb.

**Comment by SeaDve** [comment]
> Something like [this](https://tuxphones.com/tutorial-developing-responsive-linux-smartphone-apps-gnome-builder-gtk-libhandy-gtk-part-1/) is probably possible with libadwaita also. When the screen size is small, the `listen` button can go to the bottom, so that it would be easily accessible with the thumb.

Good idea, but I don't like it much since both bottom and top bar by that will be very much too "empty".

**Comment by apandada1** [comment]
A big round button in the center (instead of an almost empty bottom bar) would be nice.
Here is Deepin voice recorder.

![image](https://user-images.githubusercontent.com/3063132/130740905-7b59c461-ddcd-4cd8-b2bf-c6325d20b933.png)
